listenbrainz/webserver/views/test/test_user.py

```diff
@@ -12,7 +18,22 @@ def setUp(self):
         self.user = db_user.get_or_create('iliekcomputers')
         self.weirduser = db_user.get_or_create('weird\\user name')
 
+        self.log = logging.getLogger(__name__)
+
+        influx = InfluxDBClient(host=config.INFLUX_HOST, port=config.INFLUX_PORT, database=config.INFLUX_DB_NAME)
+        influx.query('''drop database %s''' % config.INFLUX_DB_NAME)
```

Comment:
I think these lines might not be needed here. They were put in `InfluxListenStore` tests to ensure that the database was empty at the beginning of each test run, but here since there's only one test changing the influx db, it shouldn't be a problem. Also, since we might want to clear the db after the test run, so moving this to `tearDown` might be a good idea.